docker/README.md

```diff
@@ -0,0 +1,44 @@
+This document is used for generating OWT build and all-in-one OWT running environment image.
+
+## Prequisites
+
+  * Docker is setup in your local environment and related proxy is set so that Docker can access to external network (including Google because OWT will download webrtc related code and patch from Google)
```

Comment:
```suggestion
  * Docker is setup in your local environment and related proxy is set so that Docker can access to external network (including Google because OWT will download WebRTC related code and patch from Google)
```

Reply:
Thanks for the correction, will update it.